src/Fantomas.Tests/AttributeTests.fs

```diff
@@ -30,6 +30,17 @@ let ``attributes on expressions``() =
 do ()
 """
 
+[<Test>]
+let ``attributes without parentheses on expressions``() =
+    formatSourceString false """
+    [<MyValue 55>]
+    do ()""" config
```

Comment:
Could you add a test for what happens if there are too many spaces in the attribute? (presumably they get stripped down to 1)

Reply:
Yes, will also add similar tests for args in parens